uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/ZoneValidationCookieHijackIT.java

```diff
@@ -158,55 +156,85 @@ void sessionCookieFromZone1DoesNotAllowZone2() {
         SamlIdentityProviderDefinition samlIdentityProviderDefinition1 = createTestZone1IDP(SAML_ORIGIN);
-        samlIdentityProviderDefinition1.setIdpEntityAlias(samlIdentityProviderDefinition1.getIdpEntityAlias() + "-1");
+        String originKey1 = SAML_ORIGIN + "-1";
         samlIdentityProviderDefinition1.setLinkText(LINK_TEXT_1);
         IdentityProvider<SamlIdentityProviderDefinition> provider1 = new IdentityProvider<>();
         provider1.setIdentityZoneId(zone1Id);
         provider1.setType(OriginKeys.SAML);
         provider1.setActive(true);
         provider1.setConfig(samlIdentityProviderDefinition1);
-        provider1.setOriginKey(samlIdentityProviderDefinition1.getIdpEntityAlias());
+        provider1.setOriginKey(originKey1);
         provider1.setName("simplesamlphp for testzone 1");
         provider1 = IntegrationTestUtils.createOrUpdateProvider(zoneAdminToken, baseUrl, provider1);
         assertThat(provider1.getId()).isNotNull();
 
         //we have to create two providers to avoid automatic redirect
         SamlIdentityProviderDefinition samlIdentityProviderDefinition2 = createTestZone2IDP(SAML_ORIGIN);
-        samlIdentityProviderDefinition2.setIdpEntityAlias(samlIdentityProviderDefinition2.getIdpEntityAlias() + "-2");
+        String originKey2 = SAML_ORIGIN + "-2";
         samlIdentityProviderDefinition2.setLinkText(LINK_TEXT_2);
         IdentityProvider<SamlIdentityProviderDefinition> provider2 = new IdentityProvider<>();
         provider2.setIdentityZoneId(zone2Id);
         provider2.setType(OriginKeys.SAML);
         provider2.setActive(true);
         provider2.setConfig(samlIdentityProviderDefinition2);
-        provider2.setOriginKey(samlIdentityProviderDefinition2.getIdpEntityAlias());
+        provider2.setOriginKey(originKey2);
         provider2.setName("simplesamlphp for testzone 2");
         provider2 = IntegrationTestUtils.createOrUpdateProvider(zoneAdminToken, baseUrl, provider2);
         assertThat(provider2.getId()).isNotNull();
 
         String testZone1Url = baseUrl.replace("localhost", zone1Id + ".localhost");
         String testZone2Url = baseUrl.replace("localhost", zone2Id + ".localhost");
-        webDriver.get("%s/logout.do".formatted(baseUrl));
-        webDriver.get("%s/logout.do".formatted(testZone1Url));
-        webDriver.get("%s/logout.do".formatted(testZone2Url));
-
-        // Login on zone 1
-        webDriver.get("%s/login".formatted(testZone1Url));
-        assertThat(webDriver.getTitle()).isEqualTo(zone1.getName());
-        LoginPage loginPage1 = new LoginPage(webDriver);
-        loginPage1.assertThatSamlLink_goesToSamlLoginPage(LINK_TEXT_1)
-                .assertThatLogin_goesToHomePage("marissa", "koala");
+        webDriver.get(String.format("%s/logout.do", baseUrl));
+        webDriver.get(String.format("%s/logout.do", testZone1Url));
+        webDriver.get(String.format("%s/logout.do", testZone2Url));
```

Comment:
String.format(....) and "%s/logout.do".formatted(baseUrl) are same mostly so no need to change this